### ISSUE TITLE
Planning for a virtual TPAC meeting

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/webappswg#35]
* We should find out who wants to meet for a virtual meeting?  
   * Smaller chunks of meetings.
   * Status updates - on GitHub write up like last year. See if progress has been made.   
   * Could we change it do dial in over? 
   * Maybe over 2 weeks? Or restart the calls... 
   * Maybe call them working sessions...
   * How do we deal with cross group pollination of ideas... 

Editing task force already meets monthly?

**Comment by inexorabletash** [comment]
+1 to status updates on github - that seemed to work well. Also encourage as much async discussion and decision making as possible.

Re: Working sessions: I found the "breakouts" that emerged in Fukuoka to be very productive (one on general Storage, one on Indexed DB specifically). I'd like to capture that again, while being more inclusive: i.e. having them announced/scheduled ahead of time, and not competing for space/time/attention. So: gather a list, schedule them non-overlapping to take advantage of not having a tight F2F schedule, which addresses cross-pollination - apart from time zones, there won't be scheduling conflicts. (Other than cross-WG...)

**Comment by garykac** [comment]
> [...] i.e. having them announced/scheduled ahead of time, and not competing for space/time/attention. So: gather a list, schedule them non-overlapping to take advantage of not having a tight F2F schedule

+1 +1 +1 +1 +1  I wanted to participate or listen in on more than one of the overlapping breakout sessions. The discussions also felt a bit siloed – I don't think any of the breakouts had remote participants and I don't recall a lot of scribing.

Is there anything preventing us from recording the video/audio for these sessions?

Also, I've always feel bad for the people dialing in remotely because it's often hard to follow the discussion (and side-discussions) happening in the room. It would be nice to use this opportunity to experiment with ways to improve remote participation that we could carry on once we start meeting in person again.

**Comment by marcoscaceres** [comment]
We discussed the doing the following on the call: 

- Announce that TPAC will be virtual this year - dates forthcoming. 
- Will your your community want to meet (what time? for how long?).... somehow centralize the schedule (probably just a shared online Calendar). 
- Provide a status report - with a deadline. 
- Provide a cross-group "office hours" for general chat... maybe something like a big zoom meeting and get more people on the Slack channel... or people create their own Slacks channels.

**Comment by marcoscaceres** [comment]
Fixed some typos above...

**Comment by LJWatson** [comment]
I've posted issues on the following repos:
* ARIA in HTML
* File API
* Clipboard API & Events
* Gamepad API
* IndexedDB
* Input Events Levels 1 and 2
* Image Resource
* Intersection Observer
* Pointer Lock
* Push API
* Selection API
* Screen Orientation API
* UI Events
* UI Events Key Values
* UI Events Code Values
* Web Share
* Webapp Manifest

**Comment by marcoscaceres** [comment]
Excellent, thanks @LJWatson!

**Comment by inexorabletash** [comment]
Do we want a session (or multiple sessions) on specs in incubation (e.g. in WICG) that are on track to graduating to the WebAppsWG soonish?

A non-comprehensive list of ones that I'm somewhat involved in, at different stages.

* [Web Locks](https://github.com/wicg/web-locks)
* [Cookie Store](https://github.com/WICG/cookie-store)
* [File System](https://github.com/WICG/native-file-system)
* [Keyboard Map](https://github.com/WICG/keyboard-map)

I'm certain there are many more.

FWIW, I've never been entirely clear on whether the WICG sessions at TPAC are intended to be a catch-all for anything in WICG, or a meta-discussion of the CG process itself, or both. We should avoid overlap, though! But the relevant part is at least any specs that are graduating from incubation and fit into the charter or would require rechartering.

**Comment by marcoscaceres** [comment]
> Do we want a session (or multiple sessions) on specs in incubation (e.g. in WICG) that are on track to graduating to the WebAppsWG soonish?
> ... snip...
> FWIW, I've never been entirely clear on whether the WICG sessions at TPAC are intended to be a catch-all for anything in WICG, or a meta-discussion of the CG process itself, or both.

Can be both... 

> We should avoid overlap, though! But the relevant part is at least any specs that are graduating from incubation and fit into the charter or would require rechartering.

You are on point about requiring rechartering. It would be appropriate to have a "pitching" session to pitch new deliverables to working group members/implementers, as the working group needs to recharter to adopt those. Right now, only "Cookie Store" is in scope for the working group. 

To adopt the new items, we would need interest from other browser vendors - and accompanying "[intent to migrate](https://github.com/WICG/admin/blob/gh-pages/intent-to-migrate.md)". 

However, it wouldn't be appropriate to have "working sessions" for WICG things as part of the WebAppsWG meeting... as actual technical work on the specs you listed should happen at the WICG until we adopt them. cc @wicg/chairs. 

For sessions specifically for WICG items, they are best organized in:   https://github.com/WICG/admin/issues/122

**Comment by marcoscaceres** [comment]
We set up a calendar: 
https://calendar.google.com/calendar/b/1?cid=Y19xaWlxZmJjMm9icXBvdXBwa2NiZDZyMXViMEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t

**Comment by marcoscaceres** [comment]
Rules to have a meeting: 

  * must take minutes: however, we don't mind where the minutes are taken, as long as they end up somewhere where we can collate them (e.g., google docs). 

  * We need an example of how to take good minutes.
     - Who is taking is taking the minutes 
     - who is present is present
     - who sent regrets
     - Clearly labelling Topics discussed 
     - Actions (these can just be links to GitHub Issues)
     - Any "resolutions", although asynchronous resolutions are preferred through consensus. 

We can reuse much of https://www.w3.org/2002/ws/addr/minutes.html except to for the zakim bits... people can even use IRC if they want Zakim support.

**Comment by marcoscaceres** [comment]
Potentially the least horrible:
https://www.timeanddate.com/worldclock/meetingdetails.html?year=2020&month=10&day=26&hour=21&min=0&sec=0&p1=152&p2=137&p3=179&p4=136&p5=33

**Comment by siusin** [comment]
[TPAC 2020 Group Meetings](https://www.w3.org/wiki/TPAC/2020/GroupMeetings#WG.2FIG.2FBG_Group_Meetings_details):

- Joint Group meetings - week of 12-16 October
- WG/IG/BG Group Meetings - week of 19-23 October
- the TPAC breakout sessions -  week of 26 to 30 October

I added a 2-hr event on 19 Oct (20 Oct those in AEDT and CST) to the WebApps WG TPAC calendar.

**Comment by rniwa** [comment]
I'm a bit confused here. When exactly is the meeting? Is it 2020-10-26T21:00:00Z?

**Comment by siusin** [comment]
@rniwa no, WebApps WG meeting will be on 2020-10-19
https://www.timeanddate.com/worldclock/meetingdetails.html?year=2020&month=10&day=19&hour=21&min=0&sec=0&p1=152&p2=137&p3=179&p4=136&p5=33

WG meetings will be in the week of 19-23 October to avoid conflict with the breakout sessions (week of 26 to 30 October).

**Comment by siusin** [comment]
Zoom Info -> https://lists.w3.org/Archives/Member/w3c-archive/2020Sep/0628.html

@marcoscaceres @LJWatson any update on the meeting agenda page?

**Comment by Cupruminvictus** [comment]
��i�h����^�(v��Z��
i�|��*.b����
